.github/workflows/nightly.yml

```diff
@@ -0,0 +1,104 @@
+name: Nightly
+
+on:
+  push:
+  ## Allow manual invocation
+  #workflow_dispatch:
+  #schedule:
+  #  - cron: '0 5 * * *' # UTC
```

Comment:
:information_source:  we will remove `push` and restore `schedule` for cron trigger + `workflow_dispatch` for running nightly builds on-demand